src/concurrently.spec.ts

```diff
@@ -2,6 +2,10 @@ import { ChildProcess, KillProcess, SpawnCommand } from './command';
 import { concurrently, ConcurrentlyCommandInput, ConcurrentlyOptions } from './concurrently';
 import { createFakeProcess, FakeCommand } from './fixtures/fake-command';
 import { FlowController } from './flow-control/flow-controller';
+import { Logger } from './logger';
+import { OutputWriter } from './output-writer';
+
+jest.mock('./output-writer');
```

Comment:
That's one thing I don't enjoy very much from Jest.

`jest.mock` couples the test to the implementation.
You usually don't/shouldn't care about what a function is doing internally, as long as its results are what you expected.

I think that a better way to target the output writer would be to see if something gets written to the output stream when a logger is specified in the options instead.
WDYT?

Reply:
I'm pretty new to Jest so I really appreciate your feedback here!

> That's one thing I don't enjoy very much from Jest.
> 
> `jest.mock` couples the test to the implementation. You usually don't/shouldn't care about what a function is doing internally, as long as its results are what you expected.

I fully agree with you...

> I think that a better way to target the output writer would be to see if something gets written to the output stream when a logger is specified in the options instead.
> WDYT?

That makes much more sense! I've now updated the test in #335.